examples/rtc.rs

```diff
@@ -45,7 +45,8 @@ fn main() -> ! {
 
     let serial = dp.USART2
         .usart(
-            (gpioa.pa2, gpioa.pa3),
+            gpioa.pa2,
+            gpioa.pa3,
```

Comment:
Are independent parameters better, or should we accept a `(TX, RX) where TX: TxPin, RX: RxPin` instead?

Reply:
I like the non tuple version better :slightly_smiling_face:

Reply:
I noticed that there's a precedent for tuples: https://github.com/stm32-rs/stm32f0xx-hal/blob/33fe6da6dcb7a109e6c8a0251e2f96b770a2dec0/src/serial.rs#L297 We don't necessarily have to follow that though, there are other slight differences in the implementation.

Reply:
@arkorobotics @hannobraun what do you think about this?

Reply:
Sorry for the delay. I don't have an opinion at this time. Just a vague preference for separate arguments, although I don't know why.

Reply:
Ok, let's keep it like this!